src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapperTest.java

```diff
@@ -56,7 +60,42 @@ public void should_map_zip_envelope_correctly() throws Exception {
         assertSameNonScannableItems(dbEnvelope, zipEnvelope);
         assertRightScannableItemsHaveWarnings(dbEnvelope, ocrValidationWarnings);
 
-        // properties specific to DB envelope
+        assertDbEnvelopeSpecificProperties(container, dbEnvelope);
+    }
+
+    @Test
+    public void should_map_document_types_and_subtypes_correctly() throws Exception {
+        InputEnvelope zipEnvelope = getEnvelopeFromMetafile(METAFILE_BASE_PATH + "with-scannable-items.json");
+        String container = "container1";
+
+        Envelope dbEnvelope = EnvelopeMapper.toDbEnvelope(zipEnvelope, container, Optional.empty());
+
+        assertSameOcrData(dbEnvelope, zipEnvelope);
+        assertDocumentTypes(
+            dbEnvelope,
+            CHERISHED,
+            OTHER,
+            OTHER,
+            OTHER,
+            OTHER,
+            FORM,
+            DocumentType.COVERSHEET,
```

Comment:
can be static import as other doc types above

Reply:
there is already static import of DocumentSubtype.COVERSHEET